src/util/generatePreview.js

```diff
@@ -58,6 +65,7 @@ class PreviewGenerator {
     if (options.propagateErrorsToParent) {
       this._addErrorHandling();
     }
+    this._addAlertHandling();
```

Comment:
Is this happening unconditionally? We definitely want to keep the browser default functionality when we’re generating a pop-out preview I think